pages/user/login.vue

```diff
@@ -87,26 +87,43 @@ export default defineComponent({
     const otherError = ref('')
 
     gql`
-      mutation Login($email: EmailAddress!, $password: String!) {
-        login(email: $email, password: $password) {
-          id
+      mutation LoginMutation(
+        $loginEmail: EmailAddress!
+        $loginPassword: String!
+      ) {
+        login(email: $loginEmail, password: $loginPassword) {
+          ... on UserReturned {
+            user {
+              id
+            }
+          }
+          ... on InputValidationProblem {
+            problems {
+              path
+              message
+            }
+          }
         }
       }
     `
     const {
       mutate: loginUser,
       onDone,
       error: graphqlError,
-    } = useMutation(LoginDocument, () => ({
+    } = useMutation(LoginMutationDocument, () => ({
       variables: {
-        email: email.value,
-        password: password.value,
+        loginEmail: email.value,
+        loginPassword: password.value,
       },
     }))
     const router = useRouter()
     onDone((result) => {
       if (result.data?.login) {
-        currentUserVar(result.data.login)
+        currentUserVar(
+          result.data.login.__typename === 'UserReturned'
```

Comment:
I think this check should be moved to the outer if check, and in the else statement `otherError.value` should be set to the message of the problem.